build.sbt

```diff
@@ -174,6 +173,7 @@ def exampleCommonSettings = Seq(
 )
 
 lazy val exampleAkka = (project in file("examples/akka"))
+  .disablePlugins(sbtassembly.AssemblyPlugin)
```

Comment:
NOw I get: 

```
 there are 3 keys that are not used by any other settings/tasks:
[warn]  
[warn] * exampleAkka / mainClass
[warn]   +- /Users/lg/projects/mesmer/build.sbt:198
[warn] * exampleAkkaStream / mainClass
[warn]   +- /Users/lg/projects/mesmer/build.sbt:212
[warn] * exampleZio / mainClass
[warn]   +- /Users/lg/projects/mesmer/build.sbt:226
```

@ptrdom any hints? Or is this just an sbt oddity?

Reply:
No, this is an actual problem. But we do not need that setting - those modules only have on main class in them.